tools/helper/helper.go

```diff
@@ -124,6 +124,22 @@ func CreateOperatorEnvVar(repo, deployClusterResources, operatorImage, provision
 			Name:  "PULL_POLICY",
 			Value: pullPolicy,
 		},
+		{
+			Name: "INSTALLER_PART_OF_LABEL",
+			ValueFrom: &corev1.EnvVarSource{
+				FieldRef: &corev1.ObjectFieldSelector{
```

Comment:
Same here, we want to add APIVersion field so the operator doesn't attempt to reconcile it away.

Reply:
This is the operator deployment itself so we won't be reconciling this,
should I still add v1 here for the order of things?

Reply:
Good point, not sure if it will help the HCO operator or not. But if it is a problem they will let us know.